addons/website_slides/tests/test_slide_slide.py

```diff
@@ -115,3 +115,37 @@ def test_video_vimeo(self):
                     })
                     self.assertEqual('vimeo', slide.video_source_type)
                     self.assertEqual(vimeo_id, slide.vimeo_id)
+
+    def test_compute_category_completion_time(self):
```

Comment:
You add a test in 'TestVideoFromURL' class, which is about videos URL management ... please don't add test randomly in a file, try to find the best place to put them. The 'TestSlideInternals' class is probably a better place for it :)

Reply:
Oh yes..Thank you for the review 🙏🏽 🙂